x-pack/platform/plugins/shared/lens/public/vis_type_alias.ts

```diff
@@ -39,7 +39,7 @@ export const lensVisTypeAlias: VisTypeAlias = {
       docTypes: [LENS_EMBEDDABLE_TYPE],
       searchFields: ['title^3'],
       clientOptions: { update: { overwrite: true } },
-      client: getLensClient,
+      client: getLensBasicClient,
```

Comment:
Use new basic Lens client instead of the full one that expects to receive and return the new API format.